src/plots/gl2d/scene2d.js

```diff
@@ -286,10 +292,11 @@ proto.updateFx = function(options) {
 
     fullLayout.dragmode = options.dragmode;
     fullLayout.hovermode = options.hovermode;
-};
 
-var relayoutCallback = function(scene) {
+    this.graphDiv._fullLayout = fullLayout;
```

Comment:
Here's the _cleanest_ way (I think) of updating `gd._fullLayout._plots.xy._scene2d.fullLayout`

https://github.com/plotly/plotly.js/commit/b50b7e99bb8e720240fae7c4ae0e8e3a1bb8a0ff